sw/host/opentitantool/src/main.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Result;
+use log::LevelFilter;
+use structopt::StructOpt;
+
+mod backend;
+mod command;
+use command::CommandDispatch;
+
+#[derive(Debug, StructOpt, CommandDispatch)]
+enum RootCommandHierarchy {
+    // Flattened because `Greetings` is a subcommand hierarchy.
+    #[structopt(flatten)]
+    Greetings(command::hello::Greetings),
+}
+
+#[derive(Debug, StructOpt)]
+struct Opts {
+    #[structopt(long, default_value = "off")]
+    logging: LevelFilter,
+
+    #[structopt(flatten)]
+    backend_opts: backend::BackendOpts,
+
+    #[structopt(subcommand)]
+    command: RootCommandHierarchy,
+}
+
+fn main() -> Result<()> {
+    let opts = Opts::from_args();
+
+    let mut builder = env_logger::Builder::from_default_env();
```

Comment:
I don't think you need this binding? You can just do `Builder::blah().blah().init()`

Reply:
done

Reply:
Did you forget to commit or push the changes?